src/components/dashjs-generic-modal/dashjs-generic-modal.css

```diff
@@ -1,3 +1,7 @@
 :host {
   display: block;
 }
+
+.spacing {
```

Comment:
should we use a global css class for it? Couse you are using it in `src/components/dashjs-api-control-element/dashjs-api-control-element.css` aswell